src/main/java/command/InvalidInputException.java

```diff
@@ -0,0 +1,7 @@
+package command;
+
+public class InvalidInputException extends Exception{
```

Comment:
Might want to add a space between `...extends Exception` and the curly bracket -- just a small coding style thing